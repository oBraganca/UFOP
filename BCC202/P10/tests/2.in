2
-27 71 -26 30 -20 32 -5 27 -28 -30 14 9 18 -17 -25 11 31 1 16 29 -31 17 -10 28 -16 -27 -14 -29 -35 -8 -13 24 26 -3 -12 6 10 13 -9 -24 -21 -15 23 0 22 12 -6 -7 -18 -1 20 2 4 7 -33 34 15 3 35 19 5 -23 -32 33 -11 25 -22 8 -2 -4 21 -19 -34
-45 55 20 14 16 -1 -6 12 -2 -13 27 4 -23 -14 24 -9 -27 26 17 -20 15 5 13 11 10 -19 21 1 9 -5 -18 -7 -17 -26 -8 25 22 -12 -16 3 23 18 -4 7 19 -10 -25 8 -11 0 6 -3 -15 -22 -24 -21 2
