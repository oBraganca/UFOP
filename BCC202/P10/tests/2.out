24 3
55 5
